app.js

```diff
@@ -58,10 +58,11 @@ function runNpm(command) {
 
 try {
 	require('sugar');
+	require('es6-shim');
```

Comment:
We also have some instances of new native prototype methods, like <code>String.prototype.startsWith</code>. I wouldn't really put my hands on fire for any Node version with <code>global.Set</code> also having every other ES6 feature we are using, but it should be fine.

Also, we aren't using <code>Set</code>, so in that case we would check for <code>global.Map</code>.